liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -726,6 +728,27 @@ private void configureLogging(Level logLevel, String logFile, boolean wasDefault
         }
     }
 
+    /**
+     * Configures the system output stream and error stream based on supplied global parameter.
+     */
+    private void setOutputStream(Logger logger, LogService logService) {
+        final LogOutputStream logOutputStream = LiquibaseCommandLineConfiguration.LOG_STREAM.getCurrentValue();
+        final Level level = LiquibaseCommandLineConfiguration.LOG_LEVEL.getCurrentValue();
+        if (logOutputStream != null) {
+            Arrays.stream(logger.getHandlers()).forEach(logger::removeHandler); // Unsure of impact of this statement.
+            StreamHandler streamHandler = new StreamHandler(logOutputStream.getOutputStream(), new SimpleFormatter());
+            streamHandler.setLevel(level);
+            setFormatterOnHandler(logService, streamHandler);
+            logger.addHandler(streamHandler);
+            switch (logOutputStream) {
+                case STDOUT:
+                    System.setErr(logOutputStream.getOutputStream());
+                case STDERR:
+                    System.setOut(logOutputStream.getOutputStream());
+            }
```

Comment:
This should be a method on the enum class.

Reply:
Done!